libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -107,7 +107,8 @@ class AP_GPS_SBF : public AP_GPS_Backend
         PVTGeodetic = 4007,
         ReceiverStatus = 4014,
         BaseVectorGeod = 4028,
-        VelCovGeodetic = 5908
+        VelCovGeodetic = 5908,
+        AttEuler       = 5938
```

Comment:
```suggestion
        AttEuler       = 5938,
```

Adding the comma just keeps future diffs shorter.